Makefile

```diff
@@ -16,8 +17,13 @@ doctest:
 	$(PYTHON) -c "import skimage, sys, io; sys.exit(skimage.doctest_verbose())"
 
 coverage:
-	$(PYTESTS) skimage --cov=skimage
+ifeq ($(suite),$(BENCHMARK))
+	$(PYTESTS) benchmarks --cov=skimage
```

Comment:
I think you should simply make this a new make target:
```
make benchmark_coverage
```

adding equal signs, and whatnot always complicates command line arguments.

Reply:
Done.